airflow/triggers/temporal.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, moment: datetime.datetime):
         # Make sure it's in UTC
         elif moment.tzinfo is None:
             raise ValueError("You cannot pass naive datetimes")
-        elif not hasattr(moment.tzinfo, "offset") or moment.tzinfo.offset != 0:
+        elif not hasattr(moment.tzinfo, "offset") or moment.tzinfo.offset != 0:  # type: ignore
```

Comment:
This is good as-is, but this kind of calls are better written with `getattr` instead. Since you’re accessing the value anyway, `getattr` has less overhead, and Mypy automatically let it return `Any` to bypass checks.